drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/columns/dom/listbox/MultiValueDOMElement.java

```diff
@@ -16,82 +16,59 @@
 package org.drools.workbench.screens.guided.dtable.client.widget.table.columns.dom.listbox;
 
 import com.google.gwt.dom.client.Style;
-import com.google.gwt.user.client.ui.ListBox;
+import org.drools.workbench.screens.guided.dtable.client.widget.table.columns.dom.BaseDTDOMElement;
 import org.drools.workbench.screens.guided.dtable.client.widget.table.model.GuidedDecisionTableUiCell;
+import org.gwtbootstrap3.client.ui.ListBox;
 import org.uberfire.ext.wires.core.grids.client.widget.context.GridBodyCellRenderContext;
-import org.uberfire.ext.wires.core.grids.client.widget.dom.impl.BaseDOMElement;
 import org.uberfire.ext.wires.core.grids.client.widget.grid.GridWidget;
 import org.uberfire.ext.wires.core.grids.client.widget.layer.GridLayer;
 
 /**
  * A DOMElement that can contain multiple values; e.g. a multi-select ListBox.
  */
-public abstract class MultiValueDOMElement<T, W extends ListBox> extends BaseDOMElement<T, W> {
+public abstract class MultiValueDOMElement<T, W extends ListBox> extends BaseDTDOMElement<T, W> {
 
-    private static final int HEIGHT = 16;
```

Comment:
Remove "dead" code and decide whether to limit the size of the editor to the cell size or let it _overflow_ (when "multi-select" ```ListBox``` at the moment).